.depcheckrc

```diff
@@ -1,2 +1,2 @@
-ignores: ["mocha", "rimraf", "sinon"]
+ignores: ["mocha", "rimraf", "sinon", "@types/xmldom"]
```

Comment:
This should only go in the `adaptive-expressions` `package.json` file:
```json
{
  ...,
  "scripts": {
    ...,
    "depcheck": "depcheck ... --ignores @types/xmldom",
  },
  ...
}
```

Reply:
It gives an error if the depcheckrc file is provided? I can do it but then I must remove calling of depcheckrc file from the call

Reply:
Okey my error but now i get
```
> depcheck --config ../../.depcheckrc --ignores @types/xmldom

Missing dependencies
* sinon: .\tests\expressionParser.test.js
```

Reply:
I added sinon and remove @types/xmldoc from depcheckrc. Now it should be fine